.github/workflows/test-release-publish.yml

```diff
@@ -1,8 +1,14 @@
-name: Test and publish
+name: Test and release
 on:
   push:
-    tags:
-      - '*.*.*'
+    branches:
+      - master
+      - 'hotfix-[0-9]+.[0-9]+**'
```

Comment:
do we use this hotfix tag?

Reply:
or should we :D?

Reply:
I think this goes away as now we will tag the release and push the image to the registry. We will remove this second part where we create another commit for the tag.